core/src/main/java/org/apache/james/core/MailAddress.java

```diff
@@ -239,13 +239,21 @@ public MailAddress(String address) throws AddressException {
             throw new AddressException("Addresses cannot start end with '.' or contain two consecutive dots");
         }
 
+        if (haveBackSlashComma(localPart)) {
```

Comment:
Say: not handle by javax.mail in a comment.

Reply:
Can you explain more?

Reply:
jakarta.mail and not javax now :)